test/test_other.py

```diff
@@ -14515,3 +14515,21 @@ def test_js_preprocess_pre_post(self):
     self.do_runf(test_file('hello_world.c'), 'assertions enabled\n4', emcc_args=['-sASSERTIONS=1'])
     self.do_runf(test_file('hello_world.c'), 'assertions disabled\n4', emcc_args=['-sASSERTIONS=0'])
     self.assertNotContained('#preprocess', read_file('hello_world.js'))
+
+  @with_both_compilers
+  def test_use_port_errors(self, compiler):
+    stderr = self.expect_fail([compiler, test_file('hello_world.c'), '--use-port=invalid', '-o', 'out.js'])
+    self.assertFalse(os.path.exists('out.js'))
+    self.assertContained(['Error with --use-port=invalid | invalid port name: invalid'], stderr)
```

Comment:
You don't need [] around the expected values here.

Reply:
You must have gotten unlucky :)

I think there are a few tests that actually pass lists here, but it looks like only about 5 out of 500:

```
$ git grep "assertContained(\[" test/test_other.py | wc -l
5
$ git grep "assertContained('" test/test_other.py | wc -l
479
```

Reply:
for sure :) no worries, I fixed it